src/itype.cpp

```diff
@@ -9,6 +9,12 @@
 #include <algorithm>
 #include <cmath>
 
+std::string gunmod_location::name() const
+{
+    // Yes, currently the name is just the translated id.
+    return _( _id.c_str() );
```

Comment:
I think i understood. Strings may be defined elsewhere and marked for translation there. Then at any place _() may be used.